bin/git-browse

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+if [[ $1 == "" ]]
+then
+    remote=$(git remote | head -n 1)
+else
+    remote=$1
+fi
+
+if [[ $remote == "" ]]
+then
+    echo "Remote not found"
+    exit 1
+fi
+
+remote_url=$(git remote get-url $remote)
+
+if [[ $? -ne 0 ]]
+then
+    exit $?
+fi
+
+if [[ $remote_url = git@* ]]
+then
+    url=$(echo $remote_url | sed -E -e 's/:/\//' -e 's/\.git$//' -e 's/\w*@(.*)/https:\/\/\1/')
```

Comment:
Would it be better to use `http` instead of `https`? The `http` is always supported, but `https` might not. No `https` by default looks bad, however, if the web server supports `https`, it will redirect the browser to the `https` endpoint. So the end result is the same.

Reply:
Done in second commit